src/mlte/evidence/evidence_metadata.py

```diff
@@ -40,14 +48,24 @@ def from_json(json: dict[str, Any]) -> EvidenceMetadata:
             raise RuntimeError(
                 "Cannot deserialize EvidenceMetadata, missing key 'measurement_type'."
             )
-        return EvidenceMetadata(
+
+        additional_info: Optional[str] = None
```

Comment:
Nit: ternary logic:

```python
additional_info: Optional[str] = doc["additional_info"] if "additional_info" in json else None
```

Reply:
Ok, good point. I'll add it in another branch.